### ISSUE TITLE
Keyboard Defaults to Capitalize on Type email and password

### ISSUE THREAD

**Issue by timbo1437** [EddyVerbruggen/cordova-plugin-native-keyboard#113]
The default behavior for `type="email"` and `type="password"` is to start with the capitalization off.  When adding `configureNativeKeyboard` on an input it is defaulting to capitalization on.  Code examples:

```
<input data-nativekeyboard='configureNativeKeyboard' id="email_panel" class="p-page-input-title-input" type="email" placeholder="Email Address">
```

```
  caretColor: '#777777', // default is iOS blue btw
  appearance: 'dark',
  type: 'email',
  returnKey: {
      onPress: function() { console.log('returnKey pressed'); },
      type: 'done'
    }
```

Screenshots for reference and explanation.

![cap_issue](https://user-images.githubusercontent.com/4999373/81969027-eed05680-95e2-11ea-8846-23f939a10c0b.png)

**Comment by EddyVerbruggen** [comment]
Oh wow I didn’t realize you were going to the native inputfield feature as opposed to the messenger feature of this plugin. I only ever really documented and supported the messenger stuff because it turned out very hard to implement it for all usages (think fonts and crazy custom inputfield styles). And I actually gave up on that part of the plugin once I tried making the emoji keyboard play nice. 

But if the problems you are seeing are limited to little things like the one in this bugreport then I’m willing to try and fix them.

**Comment by timbo1437** [comment]
Yeah, I noticed the documentation was heavy on the messenger feature... but there is really nothing out there that is as good as your plugin for Cordova when it comes to displaying the keyboard.

I believe most of my issues will be small things like this, I can either hack my way through or find a different solution for most things.  If I ever submit something too intensive just let me know, I will find a workaround.

I am going to be using the messenger feature as well, so its very helpful there too.
